Add Weave Cloud agents

errordeveloper opened the pull request.

lukemarsden commented on kubernetes-templates/weave-cloud/0/rancher-compose.yml:
```diff
@@ -0,0 +1,10 @@
+.catalog:
+  name: "Weave Cloud Suite"
```
I think "Tools" is a more appropriate word than "Suite".

lukemarsden commented on kubernetes-templates/weave-cloud/0/rancher-compose.yml:
```diff
@@ -0,0 +1,10 @@
+.catalog:
+  name: "Weave Cloud Suite"
+  description: "All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"
```
Maybe change:

"All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"

"Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"

lukemarsden commented on kubernetes-templates/weave-cloud/0/rancher-compose.yml:
```diff
@@ -0,0 +1,10 @@
+  version: "latest"
+  questions:
+    - variable: "WEAVE_CLOUD_SERVICE_TOKEN"
+      description: "Service authentication toke for you Weave Cloud instance, signup at https://cloud.weave.works"
```
s/toke/token

s/signup/get one

lukemarsden commented on kubernetes-templates/weave-cloud/README.md:
```diff
@@ -0,0 +1,21 @@
+# Weave Cloud Suite
+
+Weave Cloud provides all you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools.
```
=> "Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices"

lukemarsden commented on kubernetes-templates/weave-cloud/README.md:
```diff
@@ -0,0 +1,21 @@
+
+Weave Cloud provides all you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools.
+
+_To learn more and singup please visit [weave.works](https://weave.works)._
```
s/singup/sign up

rawmind0 replied:
This is not addressed correctly. 
s/sing up/sign up/

lukemarsden commented on kubernetes-templates/weave-cloud/README.md:
```diff
@@ -0,0 +1,21 @@
+
+_To learn more and singup please visit [weave.works](https://weave.works)._
+
+You will need to provide an authentication token before you install this stack.
```
You will need a service token which you can get from https://cloud.weave.works/

lukemarsden commented on kubernetes-templates/weave-cloud/config.yml:
```diff
@@ -0,0 +1,6 @@
+name: "Weave Cloud Suite"
+description: |
+  All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools
```
"Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"

lukemarsden commented on kubernetes-templates/weave-cloud/config.yml:
```diff
@@ -0,0 +1,6 @@
+description: |
+  All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools
+version: "latest"
+category: Continuous Delivery, Monitoring & Visualization
```
s/Visualization/Troubleshooting

rawmind0 commented:
Hi @errordeveloper ...Thanks so much for the cool PR .... :)

I've tested the package and is working ok. Please take a look to the @lukemarsden suggestions before merge. 

Thanks..best regards...

errordeveloper commented:
I've address the comments. Thanks @rawmind0 and @lukemarsden, PTAL.

rawmind0 commented:
Hi @errordeveloper ... Thanks for address the comments. There is one still wrong, please review y comment. 

Once of this will be addressed, it's ok.. :)

lukemarsden commented on kubernetes-templates/weave-cloud/README.md:
```diff
@@ -0,0 +1,23 @@
+
+This package contains the agents which connect your cluster to Weave Cloud.
+
+_To learn more and sing up please visit [Weaveworks website](https://weave.works)._
```
sign

rawmind0 replied:
Just change the line 7 from
+_To learn more and sing up please visit [Weaveworks website]
to
+_To learn more and sign up please visit [Weaveworks website]

lukemarsden commented:
LGTM!

errordeveloper commented:
I've rebased and fixed the type @lukemarsden pointed out. How's it looking to you now @rawmind0? I'm really unsure which comment you were referring to...

rawmind0 commented:
Hi @errordeveloper ...

the package is working well. Just correct the typo commented and i'll merge it. :)

The typo is in file  kubernetes-templates/weave-cloud/README.md line 7....

errordeveloper commented:
@rawmind0 haha, I've changed it locally and haven't amended the commit – done, thank you!

rawmind0 commented:
Thanks... @errordeveloper 

LGTM
